lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -195,27 +196,31 @@ def message_from(response)
 
     def parse_document(xml)
       begin
-        document = REXML::Document.new(xml)
-      rescue REXML::ParseException
-        return { :success => FAILURE }
+        document = Nokogiri::XML(xml)
+      rescue Nokogiri::XML::SyntaxError
+        return XML_FAILURE_RESPONSE
       end
     end
 
+    TRACKING_NUMBER_QUERY = 'FulfillmentShipmentPackage > member > TrackingNumber'.freeze
```

Comment:
When you say inline, you mean placing them at the top of class declaration or declaring multiple const in one row like

```
A, B, C = 1, 2, 3
```